Update data-classification-content-explorer.md (#3863)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/data-classification-content-explorer.md

> [!IMPORTANT]
> These permissions supercede permissions that are locally assigned to the items, which allows viewing of the content. 

> There are two roles that grant access to content explorer and it is granted using the Microsoft Security & Compliance Center(https://protection.office.com/permissions):

Reviewer: ```suggestion
There are two roles that grant access to content explorer and it is granted using the [Microsoft Security & Compliance Center](https://protection.office.com/permissions):
```

Reviewer: Apologies @msbemba - my mistake.

> There are two roles that grant access to content explorer and it can be granted using the Microsoft Security and Complaince center(https://protection.office.com/permissions0:

Reviewer: ```suggestion
There are two roles that grant access to content explorer and it is granted using the Microsoft Security & Compliance Center(https://protection.office.com/permissions):
```
